interface-definitions/service_dhcp-server.xml.in

```diff
@@ -10,12 +10,111 @@
         </properties>
         <children>
           #include <include/generic-disable-node.xml.i>
-          <leafNode name="dynamic-dns-update">
+          <node name="dynamic-dns-update">
             <properties>
               <help>Dynamically update Domain Name System (RFC4702)</help>
-              <valueless/>
             </properties>
-          </leafNode>
+            <children>
+              #include <include/dhcp/ddns-settings.xml.i>
+              <tagNode name="tsig-key-name">
+                <properties>
+                  <help>Name of the TSIG key for DNS updates</help>
```

Comment:
Maybe just "TSIG key for DNS updates"? The option name makes it clear that it's a name.